libkbfs/block_journal.go

```diff
@@ -226,33 +229,40 @@ func aggregateInfoPath(dir string) string {
 	return filepath.Join(dir, "block_aggregate_info")
 }
 
-func (j *blockJournal) changeBytes(
-	deltaStoredBytes, deltaUnflushedBytes int64) error {
+func (j *blockJournal) changeCounts(
+	deltaStoredBytes, deltaStoredFiles, deltaUnflushedBytes int64) error {
 	j.aggregateInfo.StoredBytes += deltaStoredBytes
+	j.aggregateInfo.StoredFiles += deltaStoredFiles
```

Comment:
If someone has a non-empty journal and updates to a revision that includes this commit, this could overflow on the next flush, right?  Should we just set it to 0 in that case?  (Same with bytes I guess, though that ship may have sailed.)